### PR TITLE
add README.md

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

add README.md of initialize_pose_button_panel

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: localization/util/autoware_localization_rviz_plugin/initial_pose_button_panel/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# initial_pose_button_panel
+
+## Role
+
+`initial_pose_button_panel` is the package to send a request to the localization module to calculate the current ego pose
+
+It starts calculating the current ego pose by pushing the button on Rviz, implemented as a Rviz plugin.
```
Comment (kmiya) [inline localization/util/autoware_localization_rviz_plugin/initial_pose_button_panel/README.md:7]:
```suggestion
It starts calculating the current ego pose by pushing the button on Rviz, implemented as an Rviz plugin.
```
Nit: a -> an.

> How to write an application using **an** RViz visualization widget.
http://wiki.ros.org/rviz

File: localization/util/autoware_localization_rviz_plugin/initial_pose_button_panel/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# initial_pose_button_panel
+
+## Role
+
+`initial_pose_button_panel` is the package to send a request to the localization module to calculate the current ego pose
```
Comment (kmiya) [inline localization/util/autoware_localization_rviz_plugin/initial_pose_button_panel/README.md:5]:
```suggestion
`initial_pose_button_panel` is the package to send a request to the localization module to calculate the current ego pose.
```
Nit: add a period.

File: localization/util/autoware_localization_rviz_plugin/initial_pose_button_panel/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+![initialize_button](./media/initialize_button.png)
+
+## Input / Output
+
+### Input topics
+
+| Name                                           | Type                                          | Description                                                    |
+| ---------------------------------------------- | --------------------------------------------- | -------------------------------------------------------------- |
+| `/sensing/gnss/pose_with_covariance` (default) | geometry_msgs::msg::PoseWithCovarianceStamped | initial pose with covariance to calculate the current eog pose |
```
Comment (tkimura4) [inline localization/util/autoware_localization_rviz_plugin/initial_pose_button_panel/README.md:18]: (typo)

```suggestion
| `/sensing/gnss/pose_with_covariance` (default) | geometry_msgs::msg::PoseWithCovarianceStamped | initial pose with covariance to calculate the current ego pose |
```

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**